src/heretic/r_things.c

```diff
@@ -432,7 +470,14 @@ void R_DrawVisSprite(vissprite_t * vis, int x1, int x2)
         dc_translation = translationtables - 256 +
             ((vis->mobjflags & MF_TRANSLATION) >> (MF_TRANSSHIFT - 8));
     }
-
+    // [crispy] translucent sprites
+    else if (crispy->translucency && vis->mobjflags & MF_TRANSLUCENT)
+    {
+	if ((vis->mobjflags & (MF_NOGRAVITY | MF_MISCMISSILE | MF_MISSILE) && crispy->translucency & TRANSLUCENCY_MISSILE) ||
```

Comment:
I tagged effects with MF_MISCMISSILE since some of them were neither MF_NOGRAVITY nor MF_MISSLE, and I still felt they should be translucent. If we stay with the current foreseen options, I think the flag and this whole condition might not be needed and I will remove or rework them. What do you think?

Reply:
Freshly, I'd prefer if we could get away without a second flag. But if it makes things easier, that'd be alright as well. But please add some elaborate comments if you decide to go that way.